src/onesided.jl

```diff
@@ -146,6 +146,18 @@ end
 
 # TODO: add some sort of "remote buffer": a way to specify different datatypes/counts
 
+"""
+    Get(origin, target_rank::Integer, [target_disp::Integer], win::Win)
+
+Copies data from the memory window `win` on the remote rank `target_rank` into `origin`
+(with diplacement `target_disp`) using remote memory access.
+It is strongly encouraged to use the [`Buffer`](@ref) interface by passing an
+instance of [`Buffer`](@ref) for `origin`,
+but `origin` can also be an [`AbstractArray`](@ref) or [`Ref`](@ref).
```

Comment:
Using a `Buffer` only makes a difference for custom datatypes: for standard datatypes it should be pretty efficient to create one on the fly. I would prefer this be mentioned in the `Buffer` docs, as it applies to all operations.
```suggestion
`origin` can be a [`Buffer`](@ref), or any object for which `Buffer(origin)` is defined.
```